frontend/src/component/project/Project/ProjectStatus/ProjectResources.tsx

```diff
@@ -31,14 +87,56 @@ export const ProjectResources = () => {
         [segments, projectId],
     );
 
+    const makeListItem = (
+        icon: ReactNode,
+        text: string,
+        link: string,
+        linkText: string,
+    ) => (
+        <ListItemRow>
+            <ItemContent>
+                {icon}
+                <span>{text}</span>
+            </ItemContent>
+            <Link to={link}>{linkText}</Link>
+        </ListItemRow>
+    );
+
     return (
         <Wrapper>
             <ProjectResourcesInner>
-                <h3>Project Resources</h3>
-                <span>{project.members} project member(s)</span>
-                <span>{tokens.length} API key(s)</span>
-                <span>1 SDK connection(s)</span>
-                <span>{segmentCount} project segment(s)</span>
+                <Typography variant='h3' sx={{ margin: 0 }}>
+                    Project Resources
+                </Typography>
+                <ResourceList>
+                    {makeListItem(
```

Comment:
I think more reach way would be if this is react component, which takes icon as child and all other as props. 

<ListItem text={{}} ....><UsersIcon /></ListItem>

Reply:
`<ListItem text={{}} ><UserIcon /></ListItem>`

Reply:
Yeah, that could work. I experimented a bit with it, but didn't quite find what I wanted, but lemme have another look.

Reply:
Updated in 8a325ec94bdccad059de2c705e0f21d17b633b98.